.github/workflows/build.yml

```diff
@@ -20,88 +20,51 @@ env:
   POSTGRES_DB: phnx_db
 
 jobs:
-  generate-app-files:
+  rust-setup:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
 
-      - name: Set up Flutter FVM
-        uses: kuhnroyal/flutter-fvm-config-action/setup@v3
-        with:
-          path: 'app/.fvmrc'
-
       - name: Set up Just
         uses: extractions/setup-just@v2
       - name: Set up CI
         run: just setup-ci
-      - uses: Swatinem/rust-cache@v2
-
-      - name: Integrate bridge
-        run: just frb-integrate
-
-      - name: Cache generated Rust files
-        uses: actions/cache/save@v4
-        with:
-          key: generated-${{ github.sha }}
-          path: applogic/src/frb_generated.rs
 
   rust-check-dependencies:
     runs-on: ubuntu-latest
-    needs: generate-app-files
+    needs: rust-setup
```

Comment:
Since we are not using files from this step, I think this dependency can be actually removed.

Reply:
Removed.